setup.py

```diff
@@ -63,7 +63,7 @@
     INSTALL_REQUIRES += ['cython>=0.15.1']
 
 # General Requirements
-INSTALL_REQUIRES += ['sympy>=1.6.2', 'django >= 1.8, < 1.12',
+INSTALL_REQUIRES += ['sympy==1.6.2', 'django >= 1.8, < 1.12',
```

Comment:
That's a little bit too rigid version range.

Reply:
What about `sympy>=1.6`?